build_container/CENTOS7_BUILD_STATUS.md

```diff
@@ -0,0 +1,51 @@
+You are strongly encouraged to test the produced Envoy binary on CentOS 7 yourselves to ensure that it satisfies your required functionality and operates as expected.
+
+## Version 1.21.x
+Envoy version 1.21 onwards cannot currently be compiled on CentOS 7.
+
+This is due the changes that were made to `tcp_stats.cc` during the 1.21 release and these being incompatible with the linux/tcp.h headers that are included in CentOS 7.
+
+- `glibc-headers-2.17-317.el7.x86_64.rpm`
+- `kernel-headers-3.10.0-1160.el7.x86_64.rpm`
+
+```
```

Comment:
if you add `console` to the end of the backticks itll make it a bit prettier

Reply:
should be sorted